src/System.Transactions/tests/System.Transactions.Tests.csproj

```diff
@@ -15,6 +15,10 @@
     <Compile Include="TransactionScopeTest.cs" />
     <Compile Include="AsyncTest.cs" />
     <Compile Include="EnlistTest.cs" />
+    <Compile Include="CloneTxTests.cs" />
+    <Compile Include="LTMEnlistmentTests.cs" />
+    <Compile Include="HelperFunctions.cs" />
+    <Compile Include="TestEnlistments.cs" />
```

Comment:
Unix is case sensitive in file paths.  I think all of the Unix legs are failing in CI because the files getting added here are in a path with mismatched casing, e.g. "src/system.transactions/tests/CloneTxTests.cs" instead of "src/System.Transactions/tests/CloneTxTests.cs".